hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestLocalDirAllocator.java

```diff
@@ -532,4 +533,19 @@ public void testGetLocalPathForWriteForInvalidPaths() throws Exception {
     }
   }
 
+  /**
+   * Test to check the LocalDirAllocation for the less space HADOOP-16769.
```

Comment:
nit: Maybe cut the Hadoop Jira, and write what we are doing in the test, like "Test to verify creating files using LocalDirAllocation with file size exceeding any directory capacity"...

Reply:
Ack. will do that after this PR

Reply:
Sorry didn't get this, why not in this PR? I meant to just change the Javadocs to be more descriptive of the test below.

Reply:
My bad, I misunderstood you comment something else. I will make the change.